lib/tilelivecopy.js

```diff
@@ -1,25 +1,35 @@
 var url = require('url');
 var tilelive = require('tilelive');
+var tileliveOmivore = require('tilelive-omnivore');
 var TileStatStream = require('tile-stat-stream');
 var queue = require('queue-async');
 var combiner = require('stream-combiner');
 var MigrationStream = require('./migration-stream');
 
 function tilelivecopy(srcUri, s3url, options, callback) {
+  var src_loader = tilelive.load;
   var scheme = 'scanline';
   var protocol = url.parse(srcUri).protocol;
   if (protocol === 'mbtiles:') scheme = 'list';
   if (protocol === 'omnivore:') scheme = 'pyramid';
   options.type = scheme;
+  if (options.bundle === true) {
+    src_loader = function(bundle_uri, cb) {
```

Comment:
@rclark you are absolutely right.
I changed the code back to just `tilelive.load` and stepped through it in the debugger and it works now.
Probably I had an old `tilelive-omnivore` referenced that wasn't yet aware of the multiple files parameter when I implemented this.

@GretaCB thanks for checking too.